src/emulator/download.ts

```diff
@@ -21,8 +21,11 @@ module.exports = async (name: DownloadableEmulator) => {
   fs.ensureDirSync(emulator.opts.cacheDir);
 
   const tmpfile = await downloadToTmp(emulator.opts.remoteUrl);
-  await validateSize(tmpfile, emulator.opts.expectedSize);
-  await validateChecksum(tmpfile, emulator.opts.expectedChecksum);
+
+  if (!emulator.opts.skipChecksumAndSize) {
+    await validateSize(tmpfile, emulator.opts.expectedSize);
+    await validateChecksum(tmpfile, emulator.opts.expectedChecksum);
+  }
```

Comment:
I can't comment on unchanged lines so I will comment here: how does this interact with `removeOldFiles`?  That relies on version checking but here the version will never change.

I think we want to always delete everything before unzipping the new download, right?  Otherwise we could get weird errors based on what's lingering.

Reply:
yeah I'll make it clear all versions